engine/access/rpc/backend/backend_scripts.go

```diff
@@ -291,17 +299,20 @@ func isInvalidArgumentError(scriptExecutionErr error) bool {
 
 func (b *backendScripts) compareScriptExecutionResults(
 	execNodeResult []byte,
+	execDuration time.Duration,
 	execErr error,
 	localResult []byte,
+	localDuration time.Duration,
 	localErr error,
 	blockID flow.Identifier,
 	script []byte,
+	arguments [][]byte,
 	insecureScriptHash [md5.Size]byte,
 ) {
 	// record errors caused by missing local data
 	if localErr != nil && status.Code(localErr) == codes.OutOfRange {
```

Comment:
nit: I wonder if it makes sense to create an error model for this since the logic is getting quite complex, something like: `scriptExecutionError` and then we could have  `exeErr.isOutOfRange()` and lower `exeErr.compare(err)` which would encapsulate all messy details of splitting error strings. If you think it's fine this way it's fine with me.